include/problem/MFEMProblem.h

```diff
@@ -31,11 +38,30 @@ class MFEMProblem : public ExternalProblem
                    const std::string & name,
                    InputParameters & parameters);
 
-protected:
-  std::string _input_mesh;
-  std::string _formulation;
-  int _order;
-  hephaestus::BCMap _bc_maps;
-  hephaestus::MaterialMap _mat_map;
-  hephaestus::Executioner _executioner;
+
+  void addAuxVariable(const std::string& var_type, 
+                      const std::string& var_name,
+                      InputParameters& parameters);
+
+  void setMFEMVarData(EquationSystems& esRef, 
```

Comment:
Given the direction of the copy is reversed between `setMFEMVarData` and `setMOOSEVarData`, I think the order of the arguments should be reversed between them as well, to emphasise that